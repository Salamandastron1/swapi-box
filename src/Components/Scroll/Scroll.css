.scroll-container{
  grid-column: 2 / span 1;
  grid-area: scroll;
}

