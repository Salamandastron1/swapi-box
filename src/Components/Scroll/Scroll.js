import React from 'react';
import './Scroll.css';

const Scroll = (props) => {
  return (
    <section className='scroll-container'>
    </section>
  )
}

export default Scroll;