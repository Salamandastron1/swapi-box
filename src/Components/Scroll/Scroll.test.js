import Scroll from './Scroll';
import React from 'react';
import { shallow, mount, render } from 'enzyme';

describe('Scroll', () => {

  it('should have a test', () => {
    
  })

})