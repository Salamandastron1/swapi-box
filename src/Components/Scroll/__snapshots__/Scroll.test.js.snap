// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll should match the snapshot 1`] = `
<section
  className="scroll-container"
>
  <CardContainer />
  <Crawl>
    <p />
    <h3 />
    <h4 />
  </Crawl>
</section>
`;
